<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDialog</class>
 <widget class="QDialog" name="SettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>200</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Настройки</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label_rows">
     <property name="text">
      <string>Количество строк:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="rows_input">
     <property name="placeholderText">
      <string>Введите количество строк (например, 10)</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_cols">
     <property name="text">
      <string>Количество столбцов:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="cols_input">
     <property name="placeholderText">
      <string>Введите количество столбцов (например, 10)</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="buttonLayout">
     <item>
      <widget class="QPushButton" name="save_button">
       <property name="text">
        <string>Сохранить</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancel_button">
       <property name="text">
        <string>Отмена</string>
       </property>
       <property name="shortcut">
        <string>Esc</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
